ui/app/src/components/PathNavigatorTree/PathNavigatorTree.tsx

```diff
@@ -174,6 +175,17 @@ export default function PathNavigatorTree(props: PathNavigatorTreeProps) {
     fetchingPathsRef.current = [];
   }
 
+  useMount(() => {
+    if (state) {
+      // If the state is being restoring from redux, fetchingPathsRef is being processed.
+      // Added a timeout to trigger the process again with a new fetchingPathsRef
```

Comment:
Ping me to review. Didn't quite understand from the comment.
```suggestion
      // If the state is being restored from redux, fetchingPathsRef is being processed.
      // Added a timeout to trigger the process again with a new fetchingPathsRef
```